app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
```

Comment:
school and kindergarten is very hard to solve. You have to walk into and search for (quite hidden?) wheelchair toilets and elevators? IMO this is a no go.

Reply:
Well, if you have no time to go inside a building, just don't solve this quest. Your points also apply fully to "courthouse" or "embassy", which we already support. So what do you think is a blocking difference between those and a school or a kindergarten?

Reply:
At least in my experience access to courthouse is not restricted like access into kindergarten is.

Reply:
@matkoniecz you can walk to any kindergarden, open the door and ask the staff if they have a toilet for wheelchair kids. What's the problem with that instead of going into a shop and ask the same thing.